pkg/splunk/enterprise/configuration.go

```diff
@@ -1040,10 +1040,10 @@ func validateRemoteVolumeSpec(volList []enterprisev1.VolumeSpec) error {
 		if volume.SecretRef == "" {
 			return fmt.Errorf("Volume SecretRef is missing")
 		}
-		if volume.Type == "" {
+		if isAppFramework && volume.Type == "" {
```

Comment:
nit: Please add a comment as to why this isn't required for S2 and is required for `AppRepo`.  Otherwise LGTM.